score-keeper.js

```diff
@@ -1,11 +1,3 @@
-.winner{
-color: green;
-}
```

Comment:
@ewal2 moved the `css` code to a separate file. You shouldn't mix `js` and `css` in one file.